Description: Debian-specific fixes in WORKSPACE file
 Bazel needs to download some dependencies just to properly parse the
 WORKSPACE file. We comment some load statements so that it won’t download
 the things we don’t need for building Bazel binary. Also, remove @io_bazel
 reference that was causing ccache failures.
Author: Yun Peng <pcloudy@google.com>
Origin: upstream, https://github.com/meteorcloudy/bazel/commit/1c38681aec9dee4eeeeac6196154c01e67878d65
Forwarded: not-needed
Last-Update: 2020-07-22

--- a/WORKSPACE
+++ b/WORKSPACE
@@ -442,24 +442,6 @@
     ],
 )
 
-load("@bazel_toolchains//rules:rbe_repo.bzl", "rbe_autoconfig")
-
-rbe_autoconfig(
-    name = "rbe_ubuntu1804_java11",
-    detect_java_home = True,
-    registry = "gcr.io",
-    repository = "bazel-public/ubuntu1804-bazel-java11",
-    tag = "latest",
-)
-
-rbe_autoconfig(
-    name = "rbe_ubuntu1604_java8",
-    detect_java_home = True,
-    registry = "gcr.io",
-    repository = "bazel-public/ubuntu1604-bazel-java8",
-    tag = "latest",
-)
-
 # Creates toolchain configuration for remote execution with BuildKite CI
 # for rbe_ubuntu1604.
 # To run the tests with RBE on BuildKite CI uncomment the two lines below
@@ -1104,18 +1086,6 @@
 
 stardoc_repositories()
 
-load("@io_bazel_rules_sass//:package.bzl", "rules_sass_dependencies")
-
-rules_sass_dependencies()
-
-load("@build_bazel_rules_nodejs//:index.bzl", "node_repositories")
-
-node_repositories()
-
-load("@io_bazel_rules_sass//:defs.bzl", "sass_repositories")
-
-sass_repositories()
-
 register_execution_platforms("//:default_host_platform")  # buildozer: disable=positional-args
 
 # Tools for building deb, rpm and tar files.
@@ -1157,20 +1127,6 @@
     ],
 )
 
-# Projects using gRPC as an external dependency must call both grpc_deps() and
-# grpc_extra_deps().
-load("@com_github_grpc_grpc//bazel:grpc_deps.bzl", "grpc_deps")
-
-grpc_deps()
-
-load("@com_github_grpc_grpc//bazel:grpc_extra_deps.bzl", "grpc_extra_deps")
-
-grpc_extra_deps()
-
 load("//tools/distributions/debian:deps.bzl", "debian_deps")
 
 debian_deps()
-
-load("@bazel_skylib//:workspace.bzl", "bazel_skylib_workspace")
-
-bazel_skylib_workspace()
